src/components/text-editor/examples/text-editor-with-inline-images-file-storage.tsx

```
@@ -0,0 +1,116 @@
+    ) => {
+        const imageInfo = event.detail;
+        console.log(`Image deleted: ${imageInfo.fileInfoId}`);
+
+        try {
+            // Delete image from external file storage.
+        } catch (error) {
+            console.error(
+                `Failed to delete image ${imageInfo.fileInfoId}`,
+                error,
+            );
+        }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)